framework/include/MonteCarloHybridPlayer.h

```diff
@@ -0,0 +1,32 @@
+#ifndef _MONTECARLOHYBRIDPLAYER_H
+#define _MONTECARLOHYBRIDPLAYER_H
+
+#include <vector>
+
+#include "MonteCarloPlayer.h"
+#include "MonteCarloTypes.h"
+
+#define EXPLORATION_PARAM_HYBRID 1
```

Comment:
Did you try varying this value when testing against other player implementations? I noticed that the multi-threaded CPU version did better when this number was reduced from 2 to 1. Maybe reducing this to like 0.5 or 0.25 can increase the win rate.

Reply:
for sure. I have experimented around with it a little bit and from what I have found, the more simulations you are running per iteration, then the lower you want that number. Also, the more complex the game space is, the higher you want that number. for the GPU version on Mancala, it might actually be best to set it to 0 since you will be running lots of simulations and Mancala is a very small game space.